sdk/mobile-center-crashes/src/test/java/com/microsoft/azure/mobile/crashes/CrashesTest.java

```diff
@@ -489,7 +489,7 @@ public String getMessage() {
         when(ErrorLogHelper.getStoredThrowableFile(any(UUID.class))).thenReturn(new File("."));
         when(ErrorLogHelper.getErrorReportFromErrorLog(any(ManagedErrorLog.class), any(Throwable.class))).thenCallRealMethod();
 
-        when(StorageHelper.InternalStorage.readObject(any(File.class))).thenReturn(exception);
+        when(StorageHelper.InternalStorage.readObject(any(File.class))).thenReturn(exception).thenReturn(new byte[]{}).thenReturn(exception);
```

Comment:
there's another call that to readObject that expects an exception somewhere buried in the logic (so it needs to be exception, byte array, exception, exception). The test case fails otherwise